### PR TITLE
Object storage test

### PR DESCRIPTION
**Pull request by tunniclm**

### CODE REVIEW

File: test/integration/app/spec_build.js [hunk 1]
```diff
@@ -262,4 +262,43 @@ describe('Spec option and build integration tests for app generator', function (
...
   });
 
+  describe('Web application with an object storage service is able to build', function () {
+    // Swift build is slow so we need to set a longer timeout for the test
+    this.timeout(300000);
+
+    var runContext;
+
+    var spec = {
+      appType: 'web',
```
Comment (rob-deans) [inline test/integration/app/spec_build.js]: `appType` is now only `scaffold` or `crud` might want to change that plus update the others

Comment (rob-deans) [reply test/integration/app/spec_build.js]: Adding `web: true` would make it web app

**Review by rob-deans — CHANGES_REQUESTED** [review]

File: test/integration/app/spec_build.js (hunk 1 above)
Comment (tunniclm) [reply test/integration/app/spec_build.js]: OK, I'm going to add checking into the refresh generator to validate the value of `appType`. I will do this in a separate PR, check the incorrect tests fail, then fix them.

Comment (tunniclm) [reply test/integration/app/spec_build.js]: The latest commits should address all of these points, please can you review again. Thanks

**Review by rob-deans — APPROVED** [review]

**Pull request merged**